packages/footer/src/components/HelperContent.js

```diff
@@ -13,10 +13,8 @@ const Container = styled.div`
 
   align-items: center;
 
-  > .text {
-    > :first-child {
-      margin-bottom: ${props => props.size === fontSizes.MEDIUM && "8px"};
-    }
+  > div > .text {
```

Comment:
Why don't use `.text` directly? This way I cannot add another margin-bottom to text class on or new `css` prop.